website/content/community/maintainers.md

```diff
@@ -0,0 +1,52 @@
+---
+title: Maintainers
+weight: 1
+---
+
+MetalLB was created by Dave Anderson.  We are thankful for his efforts to
+create MetalLB as a valuable contribution to the Kubernetes ecosystem.  Ongoing
+MetalLB development and maintenance is run by a team of current maintainers.
+
+MetalLB maintainers are participants in the project with the ability to approve
+changes to the MetalLB code base.  The current list of maintainers can be found
+in the [MetalLB OWNERS
+file](https://github.com/metallb/metallb/blob/main/OWNERS).
+
+## Maintainer Expectations
+
+An active maintainer should:
+
+* Actively participate in reviewing pull requests and incoming issues.  Note
+  that there are no hard rules on what is “active enough” and this is left up
+  to the judgement of the current group of maintainers.
+
+* Actively participate in discussions about design and the future of the
+  project.
+
+* Take responsibility for backports to appropriate braches for PRs they approve
+  and merge
+
+* Do their best to follow the follow all code, testing, and design conventions
+  as determined by consensus among active maintainers
+
+* Gracefully step down from their maintainership role when they are no longer
+  planning to actively participate in the project
+
+## Adding New Maintainers
```

Comment:
I'm wondering whether we want to publicly specify a process for adding and removing maintainers. Personally I'm fine with the "democratic" process described here, I'm just pointing out that in my opinion we are committed to such processes if we state them on the website, so we should be OK with that.

Reply:
It's up to you all if you're happy with this process, but I do like seeing something documented.

Reply:
I think we're good with your proposed process. @rata @daxmc99 please speak up if you have a problem with committing to this process publicly.